tests/lldp/test_lldp_syncd.py

```diff
@@ -322,6 +329,12 @@ def test_lldp_entry_table_after_reboot(
     localhost, duthosts, enum_rand_one_per_hwsku_frontend_hostname, db_instance
 ):
     duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
+
+    # Verify LLDP_ENTRY_TABLE keys match show lldp table output at the start of test
```

Comment:
There will be a reboot afterwards. All lldp_entry_keys and show_lldp_table_int_list will be checked after reboot. Will this check here still useful or it is not necessary anymore with another fix got in already?

Reply:
@wumiaont Ye, you are right, that's why I move the collection and verification after reboot, please check this PR https://github.com/sonic-net/sonic-mgmt/pull/15337/files